src/backend/libinput.rs

```diff
@@ -614,17 +614,17 @@ impl AsRawFd for LibinputInputBackend {
     }
 }
 
+/// calloop source associated with the libinput backend
+pub type LibinputSource = Generic<SourceFd<LibinputInputBackend>>;
+
 /// Binds a [`LibinputInputBackend`] to a given [`LoopHandle`].
 ///
 /// Automatically feeds the backend with incoming events without any manual calls to
 /// [`dispatch_new_events`](InputBackend::dispatch_new_events). Should be used to achieve the smallest possible latency.
 pub fn libinput_bind<Data: 'static>(
     backend: LibinputInputBackend,
     handle: LoopHandle<Data>,
-) -> ::std::result::Result<
-    Source<Generic<SourceFd<LibinputInputBackend>>>,
-    InsertError<Generic<SourceFd<LibinputInputBackend>>>,
-> {
+) -> ::std::result::Result<Source<LibinputSource>, InsertError<LibinputSource>> {
```

Comment:
Doesn't seem necessary to use the fully qualified path to `Result` here.

Reply:
Yeah indeed, it's no longer necessary after #183